src/test/java/core/basesyntax/operation/OperationsTest.java

```diff
@@ -0,0 +1,121 @@
+package core.basesyntax.operation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class OperationsTest {
```

Comment:
in this class you are using handlers, strategy selection and dao
this is not unit testing, separate testing of these classes and make sure that when you test for example dao you don't depend on anything else